pkg/cincinnati/cincinnati.go

```diff
@@ -34,19 +34,36 @@ func NewClient(id uuid.UUID, proxyURL *url.URL, tlsConfig *tls.Config) Client {
 // Update is a single node from the update graph.
 type Update node
 
+// Error is returned when are unable to get updates.
+type Error struct {
```

Comment:
maybe implement `causer` interface https://github.com/pkg/errors#retrieving-the-cause-of-an-error
so that we can use the `github.com/pk/errors` to get the cause as it's used widely in the repo.

Reply:
I'd added `Cause()` locally, but didn't commit it because there were no consumers.  You want me to add it just in case?

Reply:
it's fine for now.